classes/class_clippedcamera.rst

```diff
@@ -117,6 +117,7 @@ If ``true``, the camera stops on contact with :ref:`PhysicsBody<class_PhysicsBod
 +-----------+---------------------------+
 
 The camera's collision mask. Only objects in at least one collision layer matching the mask will be detected.
+See :ref:`physics_introduction_collision_layer_code_example` for details on how to set the collision mask.
```

Comment:
Please don't modify the generated class reference files. Instead, edit the XML files in the main repository instead and open a pull request there. See [Contribute to the class reference](https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html) for more information.

```suggestion
```